auctions/tests/model_tests.py

```diff
@@ -181,13 +181,13 @@ def test_send_mail_to_matching_askers(self, mock_send_mail):
         user = mommy.make(settings.AUTH_USER_MODEL)
 
         mock_send_mail.expects_call().with_args(
-            self._add_url('[codesy] Your ask for 50 for {url} has been met'),
+            arg.any(),
```

Comment:
Changing this to `arg.any()` means any email will pass the test. We should still assert that the first arg (i.e., the subject) is what we expect, so we know the **right** email was sent.